chore(examples): update README (#3116)
gnolang/gno: examples/README.md

# Examples

This folder showcases example Gno realms (smart contracts) and pure packages (libraries).

> These examples provide a glimpse into the potential of gno.land & Gno capabilities,

Reviewer: ```suggestion
These examples provide a glimpse into the potential of gno.land and the capabilities of Gno,
```

> while also serving as a testing set for the GnoVM.

Reviewer: ```suggestion
while also serving as a test suite for the GnoVM.
```

Pure packages and realms in this folder are pre-deployed to gno.land testnets,

> making them readily available for use on-chain. However, **there is no guarantee

Reviewer: ```suggestion
making them readily available for on-chain use. However, **there is no guarantee
```

> that code in this folder is bug-free, and should be used in accordance to this fact.**

Reviewer: ```suggestion
that the code is bug-free, so it should be used with caution and an understanding of potential risks.**
```

## Structure

This folder mimics the gno.land package path system; the "root" of the system is
the `gno.land` folder. Next, it branches out to `p/` and `r/`, which contain
pure packages and realms, respectively.

## Personal Realms & Shared Content

**Prioritizing Shared Content:** As we expand our examples and use-cases, it's
essential to prioritize shared content that benefits the broader community.
These examples serve as a foundation and reference for all users.

> **Personal Realms & Pure Packages:** We're open to personal realms, but they must

Reviewer: ```suggestion
**Personal Realms & Pure Packages:** We welcome personal realms that
```

> exemplify best practices and inspire others. To maintain our repository's

Reviewer: ```suggestion
exemplify best practices and inspire others. To maintain the organization
```

> organization, we may decline some realms. If so, consider uploading

Reviewer: ```suggestion
of the monorepo, some submissions may be declined. If so, consider uploading
```

[permissionlessly](../docs/gno-tooling/cli/gnokey/state-changing-calls.md#addpackage)

> and storing the source code on a separate repo. For higher

Reviewer: ```suggestion
and storing the source code in a separate repo. For higher
```
